packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -71,7 +71,8 @@ export function useHiddenSelect<T>(props: AriaHiddenSelectProps, state: SelectSt
   return {
     containerProps: {
       ...visuallyHiddenProps,
-      'aria-hidden': true
+      'aria-hidden': true,
+      ['data-rsp-a11y-ignore']: true
```

Comment:
This probably should not have `rsp` in the name because it's not spectrum specific, this will also affect React Aria.